python-package/mlbox/preprocessing/reader.py

```diff
@@ -3,99 +3,105 @@
 # Author: Axel ARONIO DE ROMBLAY <axelderomblay@gmail.com>
 # License: BSD 3 clause
 
-import numpy as np
-import pandas as pd
 import pickle
-from sklearn.preprocessing import LabelEncoder
 import os
-import warnings
 import time
+import numpy as np
+import pd as pd
```

Comment:
Yes, I am fixing that asap. 
I did a `Replace All` `pandas` to `pd` but didn't know the IDE was that stupid to change the import also 😂